apps/andi/lib/andi_web/live/dataset_live_view.ex

```diff
@@ -198,15 +198,29 @@ defmodule AndiWeb.DatasetLiveView do
     Enum.map(datasets, &to_view_model/1)
   end
 
-  defp to_view_model(dataset) do
+  defp view_model(dataset, nil) do
+    %{
+      "id" => dataset.id,
+      "org_title" => nil,
+      "data_title" => dataset.business.dataTitle,
+      "status" => status(dataset)
+    }
+  end
+
+  defp view_model(dataset, _organization_id) do
     %{
       "id" => dataset.id,
-      "org_title" => dataset.business.orgTitle,
+      "org_title" => dataset.organization.orgTitle,
       "data_title" => dataset.business.dataTitle,
       "status" => status(dataset)
     }
   end
 
+  defp to_view_model(dataset) do
+    org_id = dataset.organization_id
+    view_model(dataset, org_id)
+  end
```

Comment:
I believe this can be structurally cleaned up into two definitions of to_view_model using a guard to check if the organization id is null

Reply:
@ksmith-accenture, I tried to take this suggestion by doing something like:
` defp to_view_model(dataset) when is_nil(dataset.organization_id) do
    ...map here...
end`

However, that resulted in: 
`(CompileError) lib/andi_web/live/dataset_live_view.ex:201: cannot invoke remote function dataset.organization_id/0 inside guards`

Could you give an example of what you mean? (I think I may have misunderstood what you're looking for here.)

Reply:
Tim pointed out my error here- I forgot to use pattern matching! Fixed.